train/trainer.py

```diff
@@ -231,25 +232,31 @@ def make_main_scheduler(self, optimizer):
         if self.train_params.lr_scheduler_type == 'Plateau':
             sched = torch.optim.lr_scheduler.ReduceLROnPlateau
             return sched(optimizer, mode='min', verbose=False,
-                         min_lr=self.train_params.min_lr,
-                         factor=self.train_params.scheduler_factor,
-                         patience=self.train_params.patience)
+                        min_lr=self.train_params.min_lr,
```

Comment:
did a pep8 style checker suggest this, or is this a leftover from earlier?

Reply:
leftover